modules/users/data/data.yaml

```
@@ -31,15 +31,22 @@ groups:
                  'ALL = (ALL) NOPASSWD: /bin/journalctl *']
   puppet-users:
     gid: 2004
-    description: limited access on puppet111 to manage SSL certificates
-    members: [macfan]
-    privileges: ['ALL = (ALL) NOPASSWD: /root/ssl-certificate',
-                 'ALL = (ALL) NOPASSWD: /var/lib/nagios/ssl-acme']
+    description: (archived group) limited access on puppet111 to manage SSL certificates
```

```suggestion
    description: (archived group)
```
I think that this can just say archived group, that way it doesn't mention puppet111, so it doesn't have to be changed if that's ever changed, and the rest of the description is unnecessary if it's archived. Up to you though.